src/migrations/20230510113903-fix-api-token-username-migration.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+        SELECT * FROM migrations WHERE name = '/20230426110026-rename-api-token-username-field';
```

Comment:
So, this didn't work in `demo` and the reason is how we did the manual down-migration with @gardleopard: we run the down migration scripts but we didn't remove the row from the `migrations` table. Therefore, this script will believe that the migration was applied but actually, it was not.

This would have been more robust if we had checked if the column existed rather than checking if the migration was applied. Unfortunately it's now late to change this again, so we're aiming to manually re-apply this migration in `demo` :facepalm:

Reply:
Ok, we fix by running `DELETE FROM migrations WHERE name = '/20230426110026-rename-api-token-username-field'` manually